LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2020 HashiCorp, Inc.
+Copyright (c) 2024 HashiCorp, Inc.
```

Comment:
I think this LICENSE file needs to reflect BUSL text instead of MPL. Here is what [boundary has in their repo](https://github.com/hashicorp/boundary/blob/main/LICENSE).

Reply:
I did see that.  I'm not sure if we need to mention the release information that they have in the [file](https://github.com/hashicorp/boundary/blob/main/LICENSE#L7). I reached out to proj-copywrite-software and I'm waiting for confirmation